src/biotite/structure/graphics/rna.py

```diff
@@ -0,0 +1,253 @@
+        each individual bond.
+    bond_linestyle : str or ndarray, shape(n,), optional (default: None)
+        The *Matplotlib* compatible linestyle of each bond. Provide a
+        single value to set the linewidth for all bonds or an array to 
+        set the linewidth for each individual bond. By default, solid 
+        lines are used for non-pseudoknotted bonds and dashed lines are 
+        used for pseudoknotted bonds.
+    bond_color : str or ndarray, shape(n,) or shape(n,3) or shape(n,4),
+                 optional (default: 'black')
+        The *Matplotlib* compatible color of each bond. Provide a single 
+        string to set the color for all bonds or an array to set the 
+        color for each individual bond. Furthermore, the color for each 
+        individual bond can be provided as array of RGB or RGBA colors.
+    backbone_linewidth : float, optional (default: 1)
+        The linewidth of the backbone.
+    backbone_linestyle : str (default: 'solid')
+        The *Matplotlib* compatible linestyle of the backbone.
+    backbone_color : str or ndarray, shape=(3,) or shape=(4,), 
+                     dtype=float, optional (default: 'grey')
+        The *Matplotlib* compatible color of the backbone.
+    base_font : dict, optional (default: {'size': 'smaller'})
+        The *Matplotlib* compatible font of the labels denoting the type
+        of each base.
+    base_box : dict or ndarray, shape(n,), optional 
+               (default: dict(pad=0, color='white'))
+        The *Matplotlib* compatible properties of the ``FancyBboxPatch``
+        surrounding the base labels. Provide a single dictionary to
+        set the properties of all base lables or an array to set the
+        properties for each individual label.
+    annotation_positions : iterable, optional (default: None)
+        The positions of the bases to be numbered. By default every 
+        second base is annotated. Please note that while the positions 
+        in the sequence are counted from zero, they are displayed on the 
+        graph counted from one.
+    annotation_offset : int, optional (default: 8.5)
+        The offset of the annotations from the base labels.
+    annotation_font : dict, optional (default: {'size': 'smaller'})
+        The *Matplotlib* compatible font of the annotations.
+    bin_path : str, optional
+        Path of the RNAplot binary.
+    """
+
+    # Check if RNAplot is installed
+    if shutil.which(bin_path) is None:
+        raise FileNotFoundError(
+            'RNAplot is not installed at the specified location, unable to '
+            'plot secondary structure.'
+        )
+
+    # Get the unknotted base pairs
+    if pseudoknot_order is None:
+        pseudoknot_order = pseudoknots(base_pairs, max_pseudoknot_order=0)[0]
+    unknotted_base_pairs = base_pairs[pseudoknot_order == 0]
+
+    # If `bond_linewidth` is not an array, extrapolate
+    if not isinstance(bond_linewidth, np.ndarray):
```

Comment:
I think a list should also be valid for this parameter (and similar parameters), e.g. the line width might be computed using a list comprehension.